test/ReactWrapper-spec.js

```diff
@@ -502,16 +502,21 @@ describeWithDOM('mount', () => {
       expect(wrapper.find('button').length).to.equal(1);
     });
 
-    it('should support object property selectors', () => {
+    it.skip('should support object property selectors', () => {
+      class Foo extends React.Component {
+        render() {
+          return <span data-test="ref" />;
+        }
+      }
       const wrapper = mount(
         <div>
           <input data-test="ref" className="foo" type="text" />
           <input data-test="ref" type="text" />
-          <button data-test="ref" prop={undefined} />
-          <span data-test="ref" prop={null} />
-          <div data-test="ref" prop={123} />
-          <input data-test="ref" prop={false} />
-          <a data-test="ref" prop />
+          <Foo prop={undefined} />
+          <Foo prop={null} />
+          <Foo prop={123} />
+          <Foo prop={false} />
+          <Foo prop />
```

Comment:
That will likely be resolved by https://github.com/airbnb/enzyme/pull/377. Why did change this from using DOM nodes to a component, though? I think we should keep this test as-is.